### ISSUE TITLE
software and databases versions

### ISSUE THREAD

**Issue by MichaelFokinNZ** [nextgenusfs/funannotate#93]
Hi, reporting the funannotate process I am trying also to report a list of software/databases versions. Which supposed to be a good practice in bioinformatics :)

All these versions are not in default log files. It is a hassle to browse all software/db folders to get them...

Can I suggest you adding this info to predict/annotate logs or even to a separate version-log/summary...

**Comment by nextgenusfs** [comment]
`funannotate check` will get you *most* of them.  some are hard to parse....

**Comment by nextgenusfs** [comment]
I need to work on the database versions.... at some point I wanted to make a `funannotate database` command that helped you manage the resources, but it hasn't risen to the top of the priority list yet.

**Comment by MichaelFokinNZ** [comment]
cool, it helps. thank you!

**Comment by MichaelFokinNZ** [comment]
any general hints how to get database's versions?

**Comment by nextgenusfs** [comment]
EggNog is v4.5.  UniProtKB would be the date you downloaded it. MEROPS is v11. dbCAN is v5.0.  Pfam I'm less sure of, you can cross reference when you downloaded versus their website.  InterPro would be in your XML file. MiBIG is v1.3.  BUSCO is v2.

GO database is listed at top of file:
```
$ head /usr/local/share/funannotate/go.obo 
format-version: 1.2
data-version: releases/2016-12-02
date: 01:12:2016 20:32
saved-by: dph
```

**Comment by nextgenusfs** [comment]
supported in v > 1.0.0
